prow/config.yaml

```diff
@@ -332,6 +342,8 @@ tide:
     kubernetes/website: squash
     kubernetes/kubernetes-docs-ja: squash
     kubernetes/kubernetes-docs-ko: squash
+    kubernetes-client/csharp: squash
+    kubernetes-client/gen: squash
```

Comment:
what is the reason for not listing other repos under kubernetes-client here too?

Reply:
Based on my exploration of other repos' commit history, these are the only two repos that use the squash method, the rest use the default.  If the desire is to move everything to squash, that is doable via PR.  I'm also fine having these two repos use the same merge method as everything else.

My goal with this PR is to automate the state of today, and then back away and let you change as you see fit.